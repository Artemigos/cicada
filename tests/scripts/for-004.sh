while echo $xxx | grep -iqv "final"
    echo "xxx begins with $xxx"
    for xxx in wip final
        echo xxx now is $xxx
    done
done
