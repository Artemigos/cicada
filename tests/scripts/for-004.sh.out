xxx begins with 
xxx now is wip
xxx now is final
